raindrops/script.js

```diff
@@ -0,0 +1,584 @@
+const startScreen = document.getElementById('start-screen');
+const startPlayBtn = document.getElementById('start-button__play');
+const startHowToPlayBtn = document.getElementById('start-button__how-to-play');
+
+const scoreScreen = document.getElementById('score-screen');
+const scoreOKBtn = document.getElementById('score-screen__OK-button');
+const statsResult = document.getElementById('stats__result');
+const statsRight = document.getElementById('stats__right');
+const statsMistakes = document.getElementById('stats__mistakes');
+
+const displayOutput = document.getElementById('calc__display__output');
+const calcBtn = document.querySelectorAll('.btn');
+const playZone = document.getElementById('play-zone');
+const waterZone = document.getElementById('water-zone');
+
+const scoreOutput = document.getElementById('score__output');
+const waveSound = document.getElementById('wave-sound');
+const correctSound = document.getElementById('correct-sound');
+const wrongSound = document.getElementById('wrong-sound');
+const audioIcon = document.getElementById('audio-icon');
+const fullscreenIcon = document.getElementById('fullscreen-icon');
+
+const сustomizeCheckbox = document.getElementById('сustomize-checkbox');
+const customizeTable = document.getElementById('customize-table');
+const customizeFirstValueFrom = document.getElementById('first-value__from');
+const customizeFirstValueTo = document.getElementById('first-value__to');
+const customizeSecondValueFrom = document.getElementById('second-value__from');
+const customizeSecondValueTo = document.getElementById('second-value__to');
+const customizeWarning = document.getElementById('customize-warning');
+const customizeNumberValidation = Array.from(document.querySelectorAll('.value-validation'));
+const customizeOperatorValidation = Array.from(document.querySelectorAll('.operator-validation'));
+let customizeCheckedOperatorArr = [];
+
+let rainDropHeight = '70px';
+let rainDropWidth = '70px';
+let rainDropTopPosition = '-70px';
+let rainDropPreviousPosition = 0;
+
+let lastOperator = '';
+let lastFirstOperand = null;
+let lastSecondOperand = null;
+
+let isStartGame = false;
+let isHowToPlayMode = false;
+let gameScoreStartStep = 10;
+let counterRainDrops = 1;
+let counterWrongAnswers = 0;
+let counterRightAnswers = 0;
+let silentMode = false;
+
+const lowWaterLevel = '19%';
+const mediumWaterLevel = '29%';
+const maxWaterLevel = '39%';
```

Comment:
Such value can be like a config with properties.
For example:
```
const waterLevel = {
  low:  '19%',
  medium:  '29%',
  max:  '39%',
};
```
So, it's more readably and understandably.